wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Mentions/UserSearchResultsViewController.swift

```diff
@@ -158,9 +166,17 @@ final class UserSearchResultsViewController: UIViewController, KeyboardCollapseO
 
     private func resizeTable() {
         let viewHeight = view.bounds.size.height
-        let minValue = min(viewHeight, CGFloat(searchResults.count) * rowHeight)
+        let contentHeight = CGFloat(searchResults.count) * rowHeight
+        let minValue = min(viewHeight, contentHeight)
         collectionViewHeight.constant = minValue
-        collectionView.isScrollEnabled = (minValue == viewHeight)
+        collectionView.isScrollEnabled = (contentHeight > viewHeight)
+
+        if searchResults.count == 1 {
```

Comment:
I don't understand why we want a different behaviour when exactly one user is displayed. But if it's too complicated to get the same behaviour for more than one users, I'm fine for now.